projects/plugins/social/src/js/components/admin-page/index.jsx

```diff
@@ -62,8 +64,10 @@ const Admin = () => {
 		if (
 			isModuleEnabled &&
 			isUpdatingJetpackSettings &&
+			! hasRefreshedSettings.current &&
 			( isAutoConversionAvailable || isSocialImageGeneratorAvailable )
 		) {
+			hasRefreshedSettings.current = true;
 			refreshJetpackSocialSettings();
 		}
 		// We want this to run only once, when the module is enabled.
```

Comment:
It should be fine to include the other dependencies. `hasRefreshedSettings` should be stable as it's a ref, and the `is...Available` values shouldn't change either.